api.go

```diff
@@ -26,7 +27,7 @@ func (tba Client) get(path string) ([]byte, error) {
 		return nil, err
 	}
 	if resp.StatusCode != 200 {
-		return nil, errors.New("response code not 200")
+		return nil, errors.New(fmt.Sprintf("response code not 200, response code is: %d", resp.StatusCode))
```

Comment:
I feel like you could simplify this a bit: "non-200 response code %d" for example.

Reply:
Gotcha, will update